doc/reduce-traffic.md

```diff
@@ -48,3 +48,7 @@ Be reminded of the effects of this setting.
   wallet is loaded or if you use the node to broadcast transactions.
 - It makes block propagation slower because compact block relay can only be
   used when transaction relay is enabled.
+- This setting sets the flag "-walletbroadcast" to be "0", if it is currently
```

Comment:
It could probably make sense to move up this bullet point to after the second bullet point, since they are related.